pkg/grpc/user_devices.proto

```diff
@@ -56,12 +57,12 @@ message UserDevice {
   optional uint64 token_id = 3;
   optional google.protobuf.Timestamp opted_in_at = 4;
   optional bytes owner_address = 5;
-  optional uint64 aftermarket_device_token_id = 6;
+  optional uint64 aftermarket_device_token_id = 6 [deprecated = true];
```

Comment:
Good to deprecate. I think we should still populate these in case anyone is using them. rewards-api certainly is.

Reply:
Isn't it tempting to scrap all this and add `optional uint64 aftermarket_device_manufacturer_token_id`?